README.md

```diff
@@ -1 +1,46 @@
-# hw01
\ No newline at end of file
+# hw01
+title: "Hw 1"
+author: Zoe Cullison-Shimada  
+  
+# hello my name is?
+I was born and raised in **New York**.
+I am a rising **third-year**.
+I am interested in **languages**. I know _French_ and _Spanish_ but I would love to learn _Japanese_.  
+  
+# something I'd like to learn about? 
+I'd like to take a philsophy class because I've never taken one before. 
+I'd like to investigate my thoughts and carry out a creative inquiry about the world. Thus, a project of mine  
+  
+*The cultural and philsophical undertones of introductions*  
+
+* forms of introduction 
+  + verbal greetings
+  + visual greetings or eye-contact
+  + hug or handshake  
+A footnote [^1]  
+[^1]: Manifestations of introductions from my ethnographic observations  
+
+* cultural undertones
+  + customs and traditions 
+  + conventional behavior
+  + social pressure  
+A footnote [^2]  
+[^2]: My observations are from being in the field. My fieldsite was the UChicago Regenstein Library.  
+
+* philsophical undertones
+  + hospes or hospitality
+  + kindness
+  + friendship  
+A footnote [^3]  
+[^3]: Citations from Cicero  
+
+***  
+# how about an artifact that you're interested in?
+[link](www.cooking.nytimes.com)  
+
+I love _cooking_ and aspire to become a talented chef. _Cooking_ requires creativity, diligence, care, hospitality, and skill...  
+
+![yum](https://dadaeats.com/raw-vegan-gluten-free-blueberry-vanilla-cheesecake/)  
```

Comment:
Was this supposed to be a link or a picture?

Reply:
A picture, but I was having difficulty.

Reply:
Then you need to make sure the URL you use inside of `()` is a link to a picture/image file, not a full web page. For instance, if you want the first image from that page, the syntax is

```
![Yum](https://i1.wp.com/dadaeats.com/wp-content/uploads/2018/10/cheesecake.jpg?w=2000&ssl=1)
```

which appears as

![Yum](https://i1.wp.com/dadaeats.com/wp-content/uploads/2018/10/cheesecake.jpg?w=2000&ssl=1)